renderer_test.go

```diff
@@ -0,0 +1,190 @@
+package atlassianwikirenderer
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/yuin/goldmark"
+)
+
+type testCase struct {
+	input    string
+	expected string
+}
+
+func TestTaskCheckbox(t *testing.T) {
```

Comment:
This doesn't look like a valid test case, atlassian doesn't have support for checkboxes.